<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>442</width>
    <height>487</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Viqo</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="housouId_label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>放送ID lv</string>
         </property>
         <property name="buddy">
          <cstring>broadID</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="broadID">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="receive">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>受信</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="disconnect">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>切断</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="openBrowser">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="toolTip">
          <string>ブラウザで開く</string>
         </property>
         <property name="text">
          <string>O</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_3" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QComboBox" name="live_waku_list">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="auto_next_waku">
         <property name="text">
          <string>次枠追従</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_4" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLineEdit" name="submit_text"/>
       </item>
       <item>
        <widget class="QPushButton" name="submit_button">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>送信</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="is184_chk">
         <property name="text">
          <string>184</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QTextBrowser" name="one_comment_view">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
      </widget>
      <widget class="QTabWidget" name="tabWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="comment_tab">
        <attribute name="title">
         <string>コメント</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout">
         <item row="1" column="0">
          <widget class="QCheckBox" name="auto_getting_user_name_chk">
           <property name="whatsThis">
            <string/>
           </property>
           <property name="text">
            <string>自動名前取得</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="3">
          <widget class="QTreeWidget" name="comment_view">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="columnCount">
            <number>8</number>
           </property>
           <attribute name="headerMinimumSectionSize">
            <number>20</number>
           </attribute>
           <column>
            <property name="text">
             <string>No.</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>プレミアム</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>ユーザ名</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>コメント</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>日付</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>ユーザID</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>184</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>放送主</string>
            </property>
           </column>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QCheckBox" name="keep_top_chk">
           <property name="text">
            <string>カーソルを常にトップ</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>コマンド</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QScrollArea" name="command_scrollArea">
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="command">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>384</width>
              <height>216</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QWidget" name="command_newWaku_widget" native="true">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_8">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QCheckBox" name="command_newWaku_chk">
                  <property name="text">
                   <string>放送開始</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="command_newWaku">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="placeholderText">
                   <string>notify-send &quot;%newWakuName%が開始されました&quot;</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="command_comment_widget" native="true">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_6">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QCheckBox" name="command_comment_chk">
                  <property name="text">
                   <string>コメント受信時</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="command_comment">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="placeholderText">
                   <string>notify-send &quot;コメント：%comment%&quot;</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="command_beforeEnd_widget" native="true">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_7">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QCheckBox" name="command_beforeEnd_chk">
                  <property name="text">
                   <string>終了前</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="command_beforeEndMinuts_spn">
                  <property name="value">
                   <number>3</number>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label">
                  <property name="text">
                   <string>分</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="command_beforeEnd">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="placeholderText">
                   <string>notify-send &quot;この放送は%lastTime%分で終了します&quot;</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="command_nextwaku_widget" native="true">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_5">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QCheckBox" name="command_nextWaku_chk">
                  <property name="text">
                   <string>次枠移動時</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="command_nextWaku">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="placeholderText">
                   <string>firefox &quot;%wakuURL%&quot;</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="command_test_widget" native="true">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <layout class="QGridLayout" name="gridLayout_3">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item row="0" column="1">
                 <widget class="QLineEdit" name="command_test">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                 </widget>
                </item>
                <item row="0" column="2">
                 <widget class="QPushButton" name="command_test_button">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="text">
                   <string>Test</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0">
                 <widget class="QLabel" name="label_2">
                  <property name="text">
                   <string>コマンドテスト</string>
                  </property>
                  <property name="buddy">
                   <cstring>command_test</cstring>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="developtest">
               <property name="text">
                <string>PushButton</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="log_tab">
        <attribute name="title">
         <string>ログ</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QTextBrowser" name="logtext"/>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="num_audience">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>来場者数: 0</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="elapsed_time">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>00:00:00</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>442</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="file">
    <property name="title">
     <string>ファイル</string>
    </property>
    <widget class="QMenu" name="stateSave">
     <property name="title">
      <string>状態セーブ</string>
     </property>
     <addaction name="save_1"/>
     <addaction name="save_2"/>
     <addaction name="save_3"/>
     <addaction name="save_4"/>
    </widget>
    <widget class="QMenu" name="stateLoad">
     <property name="title">
      <string>状態ロード</string>
     </property>
     <addaction name="load_1"/>
     <addaction name="load_2"/>
     <addaction name="load_3"/>
     <addaction name="load_4"/>
    </widget>
    <addaction name="stateSave"/>
    <addaction name="stateLoad"/>
   </widget>
   <widget class="QMenu" name="edit">
    <property name="title">
     <string>編集</string>
    </property>
    <addaction name="clear"/>
    <addaction name="separator"/>
    <addaction name="setting"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>放送</string>
    </property>
    <addaction name="autoNewWakuChk"/>
    <addaction name="getNewWakuNow"/>
    <addaction name="autoNewWakuSettings"/>
   </widget>
   <addaction name="file"/>
   <addaction name="edit"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="setting">
   <property name="text">
    <string>設定...</string>
   </property>
  </action>
  <action name="clear">
   <property name="text">
    <string>コメント一覧を消去</string>
   </property>
  </action>
  <action name="oneCommentActionSearchByGoogle">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>googleで検索</string>
   </property>
  </action>
  <action name="oneCommentActionCopy">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>コピー</string>
   </property>
   <property name="toolTip">
    <string>コピー</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="save_1">
   <property name="text">
    <string>1 : デフォルト</string>
   </property>
  </action>
  <action name="save_2">
   <property name="text">
    <string>2 :</string>
   </property>
  </action>
  <action name="save_3">
   <property name="text">
    <string>3 :</string>
   </property>
  </action>
  <action name="load_1">
   <property name="text">
    <string>1 : デフォルト</string>
   </property>
  </action>
  <action name="load_2">
   <property name="text">
    <string>2 :</string>
   </property>
  </action>
  <action name="load_3">
   <property name="text">
    <string>3 :</string>
   </property>
  </action>
  <action name="save_4">
   <property name="text">
    <string>4 :</string>
   </property>
  </action>
  <action name="load_4">
   <property name="text">
    <string>4 :</string>
   </property>
  </action>
  <action name="autoNewWakuChk">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>次枠自動取得する</string>
   </property>
  </action>
  <action name="autoNewWakuSettings">
   <property name="text">
    <string>次枠設定...</string>
   </property>
  </action>
  <action name="getNewWakuNow">
   <property name="text">
    <string>今すぐ取得</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>auto_next_waku</sender>
   <signal>toggled(bool)</signal>
   <receiver>command_nextwaku_widget</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>432</x>
     <y>90</y>
    </hint>
    <hint type="destinationlabel">
     <x>100</x>
     <y>369</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>command_comment_chk</sender>
   <signal>toggled(bool)</signal>
   <receiver>command_comment</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>152</x>
     <y>300</y>
    </hint>
    <hint type="destinationlabel">
     <x>395</x>
     <y>303</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>command_nextWaku_chk</sender>
   <signal>toggled(bool)</signal>
   <receiver>command_nextWaku</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>134</x>
     <y>366</y>
    </hint>
    <hint type="destinationlabel">
     <x>395</x>
     <y>369</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>one_comment_view</sender>
   <signal>copyAvailable(bool)</signal>
   <receiver>oneCommentActionSearchByGoogle</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>195</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>one_comment_view</sender>
   <signal>copyAvailable(bool)</signal>
   <receiver>oneCommentActionCopy</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>195</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>command_beforeEnd_chk</sender>
   <signal>toggled(bool)</signal>
   <receiver>command_beforeEnd</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>104</x>
     <y>333</y>
    </hint>
    <hint type="destinationlabel">
     <x>395</x>
     <y>336</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>command_newWaku_chk</sender>
   <signal>toggled(bool)</signal>
   <receiver>command_newWaku</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>267</y>
    </hint>
    <hint type="destinationlabel">
     <x>251</x>
     <y>270</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>menu_triggered(QAction*)</slot>
 </slots>
</ui>
